roles/matrix-bot-maubot/defaults/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+matrix_bot_maubot_enabled: true
+matrix_bot_maubot_container_image_self_build: false
+matrix_bot_maubot_docker_repo: "https://mau.dev/maubot/maubot.git"
+matrix_bot_maubot_docker_src_files_path: "{{ matrix_bot_maubot_base_path }}/docker-src"
+
+matrix_bot_maubot_version: latest
+matrix_bot_maubot_docker_image: "dock.mau.dev/maubot/maubot:{{ matrix_bot_maubot_version }}"
+matrix_bot_maubot_docker_image_force_pull: "{{ matrix_bot_maubot_docker_image.endswith(':latest') }}"
+
+matrix_bot_maubot_base_path: "{{ matrix_base_data_path }}/maubot"
+matrix_bot_maubot_data_path: "{{ matrix_bot_maubot_base_path }}/data"
+matrix_bot_maubot_config_path: "{{ matrix_bot_maubot_base_path }}/config"
+matrix_bot_maubot_container_data_dir: "/data"
+matrix_bot_maubot_container_config_dir: "/root/.config/"
+
+matrix_bot_maubot_bot_server: "https://{{ matrix_server_fqn_matrix }}"
+matrix_bot_maubot_proxy_management_interface: false
+matrix_bot_maubot_expose_management_interface: true
+
+
+matrix_bot_maubot_logging_level: info
```

Comment:
This variable seems to be unused.

```suggestion
```

Feel free to add variables for the different loggers instead though.

Reply:
Done (removing)